interface/HHAnalyzer.h

```diff
@@ -162,6 +162,14 @@ class HHAnalyzer: public Framework::Analyzer {
 
         BRANCH(gen_ttbar_decay_type, char); // Type of ttbar decay. Can take any values from TTDecayType enum
 
+        // Di-higgs gen system
+        BRANCH(gen_iH1, char);
+        BRANCH(gen_iH2, char);
+        BRANCH(gen_H1, LorentzVector);
+        BRANCH(gen_H2, LorentzVector);
+        BRANCH(gen_mHH, double);
+        BRANCH(gen_costhetastar, double);
+
```

Comment:
Ah yess indeed... Is it really an issue? It's a bit an overkill to fetch the branch for each event (internally you look for the key in a map), but I can easily move it, tell me :)